dali/base/dadfs.cpp

```diff
@@ -11838,9 +11838,23 @@ SecAccessFlags CDistributedFileDirectory::getFDescPermissions(IFileDescriptor *f
                 for (unsigned k=0;k<localpath.length();k++)
                     if ((localpath.charAt(k)=='?')||(localpath.charAt(k)=='*'))
                         localpath.setCharAt(k,'_');
-                CDfsLogicalFileName dlfn;
-                dlfn.setExternal(rfn.queryEndpoint(),localpath.str());
-                SecAccessFlags perm = getScopePermissions(dlfn.get(),user,auditflags);
+
+                SecAccessFlags perm = SecAccess_None;
+                StringBuffer planeName;
+                fdesc->getPlaneName(planeName);
+                if (!planeName.isEmpty())
+                {
```

Comment:
It doesn't make sense to be doing this inside a loop that is going through the INode's in the the file descriptor..

CDistributedFileDirectory::getFDescPermissions itself should be considered legacy, nothing should really be checking each node of a group like this.
And checkPhysicalFilePermissions in dfurun should probably not be using an IFileDescriptor as a basis to check either.
When importing and exporting it [should] know the plane (or host/ip if allowed through because isDropZoneRestrictionEnabled=false), and the logical file, such that it can ldap scope check ~plane::<planename>::logical file, or ~file::<host>::path.
NB: as noted in the JIRA it should continue to also check the old form ~file::<host>::ip (with comments to say legacy, for backward compatibility).